src/lib/EtherscanLink.svelte

```diff
@@ -1,9 +1,13 @@
-<script lang="ts">
-    import { NETWORK } from "./config";
+<script context="module" lang="ts">
+    export const config: {
+        etherscan: string | null;
```

Comment:
Same here, I think this component should not depend on the context. Probably the only component that should depend on the context is the `page` component and then pass all the context values down to the child components as props.

Reply:
Not here. The I wouldn't use `etherscanPrefix` as a component prop, because this should be the same for every instance. That's why it's at the `module` level.